applications/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1115 @@
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition name="NXstress" extends="NXobject" type="group" category="application"
+    xmlns="http://definition.nexusformat.org/nxdl/3.1"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <symbols>
+        <symbol name="n_X">
+            <doc>Number of diffractogram channels.</doc>
+        </symbol>
+        <symbol name="n_D">
+            <doc>Number of diffractograms. For example the number of energy-dispersive detectors or the number of azimuthal sections in an area detector.</doc>
+        </symbol>
+        <symbol name="n_Peaks">
+            <doc>Number of reflections.</doc>
+        </symbol>
+        <symbol name="x_Unit">
+            <doc>Diffractogram X units.</doc>
+        </symbol>
+        <symbol name="y_Unit">
+            <doc>Diffractogram Y units.</doc>
+        </symbol>
+        <symbol name="c_Unit">
+            <doc>Converted diffractogram X units (could be the same as *x_Unit*).</doc>
+        </symbol>
+        <symbol name="n_Temp">
+            <doc>number of temperatures</doc>
+        </symbol>
+		<symbol name="n_sField">
+            <doc>number of values in applied stress field</doc>
+        </symbol>
+    </symbols>
+    <doc>
+    Application definition for stress and strain analysis of crystalline material defined by the `EASI-STRESS consortium &lt;https://easi-stress.eu&gt;`_.
+
+    When a crystal is loaded (applied or residual stress) its crystallographic parameters change.
+
+    Stress and strain analysis calculates deformation (strain) and the associated force (stress)
+    from diffraction data.
+
+    This application definition essentially standardizes the result of diffraction pattern analysis
+    from different types of diffraction experiments for the purpose of stress and strain analysis.
+    The analysis is typically some form of diffraction peak indexing and fitting.
+    The experiments are for example
+
+    - energy-dispersive X-ray powder diffraction
+    - angular-dispersive X-ray powder diffraction
+    - angular-dispersive neutron powder diffraction
+    - time-of-flight (TOF) neutron powder diffraction.
+
+    In addition, the application definition guarantees that the information about instrumental setups, measurement conditions, and data analysis workflows are described.
+    This ensures not only the reproducability and tracability of the measured data, but also the meta-data. Since not all participating beamlines or instruments can provide an input to all the NXfields listed in this application definition, not all of them are "required".

```

Comment:
```suggestion
    This ensures not only the reproducability and tracability of the measured data, but also the metadata. Since not all participating beamlines or instruments can provide an input to all the NXfields listed in this application definition, not all of them are "required".

```